lib/themes/material_demo_theme_data.dart

```diff
@@ -7,29 +7,53 @@ import 'package:flutter/material.dart';
 
 class MaterialDemoThemeData {
   static final themeData = ThemeData(
-    colorScheme: _colorScheme,
-    appBarTheme: AppBarTheme(
-      color: _colorScheme.primary,
-      iconTheme: IconThemeData(color: _colorScheme.onPrimary),
-    ),
-    bottomAppBarTheme: BottomAppBarTheme(
-      color: _colorScheme.primary,
-    ),
-    canvasColor: _colorScheme.background,
-    toggleableActiveColor: _colorScheme.primary,
-    highlightColor: Colors.transparent,
-    indicatorColor: _colorScheme.onPrimary,
-    primaryColor: _colorScheme.primary,
-    backgroundColor: Colors.white,
-    scaffoldBackgroundColor: _colorScheme.background,
-    snackBarTheme: const SnackBarThemeData(
-      behavior: SnackBarBehavior.floating,
-    ),
-    typography: Typography.material2018(
-      platform: defaultTargetPlatform,
-    ),
-    visualDensity: VisualDensity.standard,
-  );
+      colorScheme: _colorScheme,
```

Comment:
nit: could you sort and keep the component themes together?

Reply:
Love it, so nice. Done.

Reply:
Right? I almost want to propose a `ThemeData` ordering lint.